pom.xml

```diff
@@ -1012,6 +1012,8 @@
                         <staticAfter>false</staticAfter>
                         <skip>${format.skip}</skip>
                         <removeUnused>true</removeUnused>
+                        <!-- Fixes impsort for JDK 19+, see https://github.com/revelc/impsort-maven-plugin/pull/83 -->
+                        <ignoreParseErrorsBelowImports>true</ignoreParseErrorsBelowImports>
```

Comment:
@yrodiere I wanted to start a build of this PR as there's a new version of impsort published and I've merged it in, but now there's a merge conflict 🙈 🙈 🙈 sorrrrry 🙈.
I've also noticed this comment: 
https://github.com/revelc/impsort-maven-plugin/blob/384083158a635353b4700a7a3ad2243aefe432be/src/main/java/net/revelc/code/impsort/maven/plugin/AbstractImpSortMojo.java#L246-L252

we probably have to remove this property 😕 as we keep the `removeUnused` ... but it should be ok since the build now passes on newer JDKs

Reply:
Rebased and removed `ignoreParseErrorsBelowImports`, thanks.